lib/opentelemetry-proto/src/convert.rs

```diff
@@ -78,69 +80,136 @@ fn kv_list_into_value(arr: Vec<KeyValue>) -> Value {
     )
 }
 
-impl From<ResourceLog> for Event {
-    fn from(rl: ResourceLog) -> Self {
-        let mut le = LogEvent::default();
+impl ResourceLog {
```

Comment:
Noting if there are any eager eyes looking at this, the following code is a draft first pass at supporting the namespaces and will be cleaned up/condensed.

Reply:
comment still valid? or has cleanup already happened

Reply:
Some cleanup happened, there's probably cleaner ways of doing some of the actions (like the initial log creation is probably messier than it needs to be). I think it's cleaned up enough for merging, unless one of y'all reviewers sees an opportunity for cleaning up/making clearer/etc.